pkg/ebpf/tracee.go

```diff
@@ -266,6 +281,16 @@ func New(cfg Config) (*Tracee, error) {
 	}
 	t.containers = c
 
+	t.networkInfo.ifacesIndexList = make(map[int]*net.Interface)
+	for _, iface := range appendStr2Str(t.config.Filter.InterfaceToTrace, t.config.Capture.NetIfaces) {
```

Comment:
maybe `appendStr2Str` is redundent now. you can just loop over these two lists and check whether the `netIface.Index` exists in the `t.networkInfo.ifacesIndexList` map (or even just update the map without checking). 
the question is what costs more - calling `net.InterfaceByName(iface)` (maximum of 2 calls per interface), or calling `appendStr2Str`.